platform/ios/PolyPodApp/PolyPod/HomeScreenView.swift

```diff
@@ -89,7 +91,9 @@ final class HomeScreenViewModel: ObservableObject {
 
     func setup() {
         storageCancellable = storage.categoriesList.sink { sections in
-            self.sections = sections
+            self.sections = sections.filter {
+                section in section.visible ?? true
```

Comment:
Wasn't quite sure where to put this logic, that if the `visible` parameter isn't specified, it's supposed to show up. Decided to put it in the view model in the end, cause that seemed most appropriate, but I'm not very attached to this.

Reply:
You can filter out the category in `importFeatures` function, where metaCategory is mapped to a FeatureCategoryModel.

Reply:
Yeah, good idea, that's cleaner.

Reply:
Done!